code/modules/client/client procs.dm

```diff
@@ -392,6 +397,13 @@
 /client/proc/InitClient()
 	to_chat(src, "<span class='alert'>If the title screen is black, resources are still downloading. Please be patient until the title screen appears.</span>")
 
+	var/local_connection = (config.auto_local_admin && (isnull(address) || localhost_addresses[address]))
+	// Automatic admin rights for people connecting locally.
+	// Concept stolen from /tg/ with deepest gratitude.
+	// And ported from Nebula with love.
+	if(local_connection && !admin_datums[ckey])
```

Comment:
Check if the permissions api is used and disable it (automatically) in that case aswell